payments-core/src/main/java/com/stripe/android/Stripe.kt

```diff
@@ -1672,20 +1675,50 @@ class Stripe internal constructor(
      * @param stripeAccountId Optional, the Connect account to associate with this request.
      * By default, will use the Connect account that was used to instantiate the `Stripe` object, if specified.
      * @param callback a [ApiResultCallback] to receive the result or error
+     * @param activity the owning activity. This will be used to identify fraud via a passive (invisible) hCaptcha
      */
     @UiThread
     @JvmOverloads
     fun createRadarSession(
         stripeAccountId: String? = this.stripeAccountId,
-        callback: ApiResultCallback<RadarSession>
+        callback: ApiResultCallback<RadarSession>,
+        activity: AppCompatActivity? = null
```

Comment:
This is a new requirement for anyone calling this endpoint. I have some mild FUD with adding this, but it's required by hCaptcha, so we may not have a choice.

Reply:
this isn't something we can obtain programmatically?

Reply:
Unfortunately because we don't know the context in which this method is called, no. I'll revisit how the SDK actually uses this and see if we can omit it.